presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4764,6 +4764,15 @@ public void testFilterPushdownWithAggregation()
         assertQuery("SELECT * FROM (SELECT count(*) FROM orders) WHERE null");
     }
 
+    @Test
+    public void testGroupByWithConstants()
+    {
+        assertQuery("SELECT * FROM (SELECT regionkey, col, count(*) FROM (SELECT regionkey, 'bla' as col from nation) group by regionkey, col)");
```

Comment:
You might want to add the expected one as well - for example:
```
select 'blah', * from (elect regiokey, count(*) from nation group by regionkey)
```

So it is more obvious.